infra/ibmcloud/terraform/k8s-s390x-infra-setup/providers.tf

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+locals {
+  key    = var.ibmcloud_api_key
+  region = "eu-de"
+  zone   = "eu-de-1"
```

Comment:
Since you’ve defined a variable for the zone, shouldn’t this value come from it?